src/views/InvolvementProfile/components/Membership/components/AdminCard/index.jsx

```diff
@@ -90,7 +88,11 @@ const AdminCard = ({ createSnackbar, isSiteAdmin, involvementDescription, onAddM
   return (
     <>
       <Card>
-        <CardHeader title="Manage Roster" style={headerStyle} />
+        <CardHeader
+          title="Manage Roster"
+          style={headerStyle}
```

Comment:
You shouldn't have both a style and a className tag.  If you have this, this is a hint that you need to move some more tags into the external CSS.